odxtools/decodestate.py

```diff
@@ -24,6 +24,12 @@ class DecodeState:
     #: (if not explicitly specified by the object to be decoded.)
     cursor_position: int = 0
```

Comment:
Maybe add byte there to reduce confusion?
```suggestion
    cursor_byte_position: int = 0
```

Reply:
I am just adding `byte` there to be able to tell it apart from the "bit" cursor position, no relation to the `byte_position` attribute is intended.

Reply:
okay, changed. I'm not happy about the length of the name of this attribute, but I can see your point...